es6/import-a-default-export.js

```diff
@@ -0,0 +1,4 @@
+import subtract from "./math_functions.js"
+// Only change code above this line
+
+subtract(7,4);
```

Comment:
Please include a space char between , and the next function parameter.
Pay attention to this because it's not the first time we talk about that